manager/README.md

```diff
@@ -142,15 +142,12 @@ To start the service, execute the following shell script (note a server needs to
 
 ```bash
 # download the latest version of the service
-go get github.com/ultravioletrs/cocos
+git clone git@github.com:ultravioletrs/cocos.git
 
-cd $GOPATH/src/github.com/ultravioletrs/cocos
+cd cocos
 
 # compile the manager
-make manager
```

Comment:
We should be capable to make Manager with `make manager`

Reply:
This has been udated